notebooks/RAG_zephyr_langchain.ipynb

```diff
@@ -0,0 +1,11589 @@
+{
```

Comment:
<ul><li>The image seems broken, also on the GH branch.</li><li><a href="https://github.com/deepset-ai/haystack" target="_blank">Haystack</a> separates its RAG systems as two parts: Retriever is the system that gathers supporting documents (with the embedding model), Reader is the LLM that builds an output for you. This separation could be used in your MLOps image?</li><li>Maybe you could add more text formatting and/or smileys to help the reading?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
* Fixed the image (The PR needed to be merged into the dataset on hub)
* Once we start publishing notebooks, we should invite Haystack to add their notebook as well for more open-source representation, and more diversity of approaches. 
* I've added a bit more structure to the text, but I'm not a massive fan of smileys in the guides (personal preference - feel free to add smileys in your notebooks!).